administrator/com_joomgallery/sql/updates/mysql/4.0.0.sql

```diff
@@ -171,8 +178,6 @@ CREATE TABLE IF NOT EXISTS `#__joomgallery_configs` (
 `jg_category_view_caption_align` VARCHAR(25) NOT NULL DEFAULT "center",
 `jg_category_view_images_show_title` TINYINT(1) NOT NULL DEFAULT 1,
 `jg_category_view_title_link` VARCHAR(25) NOT NULL DEFAULT "defaultview",
-`jg_category_view_lightbox_image` VARCHAR(25) NOT NULL DEFAULT "detail",
-`jg_category_view_lightbox_thumbnails` TINYINT(1) NOT NULL DEFAULT 0,
```

Comment:
The file 4.0.0.sql may not be changed.
When updating from JG 3, both the 4.0.0.sql and the 4.1.0.sql are processed, which then leads to the error:
```
JInstaller: :Install: SQL-Fehler Unknown column 'jg_category_view_lightbox_image' in '#_joomgallery_configs'
Extension installation stopped.
```